pkg/providers/vsphere/vmlifecycle/create_contentlibrary.go

```diff
@@ -60,6 +62,36 @@ func deployOVF(
 	return vcenter.NewManager(restClient).DeployLibraryItem(vmCtx, item.ID, deploy)
 }
 
+func deployISO(
+	vmCtx pkgctx.VirtualMachineContext,
+	vimClient *vim25.Client,
+	item *library.Item,
+	createArgs *CreateArgs) (*vimtypes.ManagedObjectReference, error) {
+
+	vmCtx.Logger.Info("Deploying ISO Library Item", "itemID", item.ID, "itemName", item.Name)
```

Comment:
Seems like `item` is not used in this function besides logging its ID and name. Should we consider dropping it and also renaming the function to something like `deployVM/createVM` as there is nothing ISO-specific involved in this function?